cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Wait for the database to be available...")
+        db_connect = None
+        while not db_connect:
+            try:
+                db_connect = connections["default"]
```

Comment:
Here, you're just assigning the connection object to `db_connect` but not actually checking if the connection to the database is available. You should use the `ensure_connection` method of the connection object to make sure the database is available. Here is how you can do it: `db_connect = connections['default'].ensure_connection()`.